JS/Lesson15/sinan/script.js

```diff
@@ -105,7 +104,21 @@ checkAge(18);
 checkAge(65);
 
 // 3. Write a program that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
-
+function checkGreater(numberFirst,NumberSecond) {
```

Comment:
camelCase is commonly used in JS to name variables, so it would be good to rename 'NumberSecond' into 'numberSecond'.

Also, it would make your code more readable if you rename the following:
- 'checkGreater' to 'compareNumbers'
- 'numberFirst' to 'number1' or 'firstNumber'
- 'NumberSecond' to 'number2' or 'secondNumber'

Remember to update all occurrences when changing the name!